Move definition of ABS() where it is needed for gcc-4.7.
http://projects.blender.org/scm/viewvc.php?view=rev&root=bf-blender&revision=44000

--- intern/elbeem/intern/utilities.h~	2011-08-11 18:39:29.000000000 +0200
+++ intern/elbeem/intern/utilities.h	2012-06-04 14:49:48.501421430 +0200
@@ -177,12 +177,6 @@
 MAX( T a, T b )
 { return (a < b) ? b : a ; }
 
-/* absolute value */
-template < class T >
-inline T
-ABS( T a )
-{ return (0 < a) ? a : -a ; }
-
 /* sign of the value */
 template < class T >
 inline T
--- intern/elbeem/intern/ntl_vector3dim.h~	2011-08-11 18:39:29.000000000 +0200
+++ intern/elbeem/intern/ntl_vector3dim.h	2012-06-04 14:50:30.542227296 +0200
@@ -113,6 +113,11 @@
 #endif // !NOGUI
 #endif // ELBEEM_PLUGIN==1
 
+/* absolute value */
+template < class T >
+inline T
+ABS( T a )
+{ return (0 < a) ? a : -a ; }
 
 // basic inlined vector class
 template<class Scalar>
